src/main/scala/com/microsoft/ml/spark/core/utils/ClusterUtil.scala

```diff
@@ -11,29 +11,29 @@ import org.apache.spark.sql.{Dataset, SparkSession}
 import org.slf4j.Logger
 
 object ClusterUtil {
-  /** Get number of cores from dummy dataset for 1 executor.
+  /** Get number of workers from dummy dataset for 1 executor.
     * Note: all executors have same number of cores,
     * and this is more reliable than getting value from conf.
     * @param dataset The dataset containing the current spark session.
-    * @return The number of cores per executor.
+    * @return The number of workers per executor.
     */
-  def getNumCoresPerExecutor(dataset: Dataset[_], log: Logger): Int = {
+  def getNumWorkersPerExecutor(dataset: Dataset[_], log: Logger): Int = {
```

Comment:
maybe getNumTasksPerExecutor might be a better name?

Reply:
Oh, when I write reply, you already commented about it :)